src/simulation/m_rhs.fpp

```diff
@@ -675,7 +675,10 @@ contains
         call nvtxEndRange
 
         call nvtxStartRange("RHS-ELASTIC")
-        if (hyperelasticity) call s_hyperelastic_rmt_stress_update(q_cons_qp%vf, q_prim_qp%vf)
+        if (hyperelasticity) then
+            call s_hyperelastic_rmt_stress_update(q_cons_qp%vf, q_prim_qp%vf)
+            call s_populate_variables_buffers(q_prim_qp%vf, pb, mv)
```

Comment:
can you confirm/deny: this is doing two rounds of communication (doubling communication time). the reason appears to be that you need to get the variables in the primitive region in order to compute _other_ variable (related to RMT) in the buffer region. those RMT variables in the buffer region are computed in `s_hyperelastic_rmt_stress_update` (throughout the domain). is this correct?

Reply:
Yes, that is correct. We need to populate the elastic stress variables to the buffer and boundary regions in preparation for the riemann_solver.

If this is undesired, we would need to find write code very similar to `s_populate_variable_buffers` to handle this. If this is preferred, let me know and I can begin to write such code.

Reply:
we can leave as is, thanks!